book/developers/exex/exex.md

```diff
@@ -2,17 +2,27 @@
 
 ## What are Execution Extensions?
 
-Execution Extensions allow developers to build their own infrastructure that relies on Reth
+Execution Extensions (or ExExes, for short) allow developers to build their own infrastructure that relies on Reth
 as a base for driving the chain (be it [Ethereum](../../run/mainnet.md) or [OP Stack](../../run/optimism.md)) forward.
 
 An Execution Extension is a task that derives its state from changes in Reth's state.
 Some examples of such state derivations are rollups, bridges, and indexers.
 
+They are called Execution Extensions because the main trigger for them is the execution of new blocks (or reorgs of old)
+initiated by Reth.
+
 Read more about things you can build with Execution Extensions in the [Paradigm blog](https://www.paradigm.xyz/2024/05/reth-exex).
 
+## What Execution Extensions are not?
+
+Execution Extensions are not separate processes that connect to the main Reth node process.
+Instead, ExExes are compiled into the same binary as Reth, and run alongside it, sharing the memory.
+
+If you want to build an Execution Extension that sends data into a separate process, check out the [Sink](./sink.md) chapter.
```

Comment:
is sink the correct term? in my mind sinks collect something, this example will send something out of process. i guess *technically* it also collects events, so really its a duplex. maybe we should just name it remote or something